instrumentation/src/main/java/io/opentelemetry/android/config/OtelRumConfig.java

```diff
@@ -23,7 +27,9 @@ public class OtelRumConfig {
     private boolean includeScreenAttributes = true;
     private DiskBufferingConfiguration diskBufferingConfiguration =
             DiskBufferingConfiguration.builder().build();
+
     private boolean networkChangeMonitoringEnabled = true;
+    private final List<InitializationListener> initializationListeners = new ArrayList<>();
```

Comment:
Should this list be thread-safe or not expected/allowed to be called from multiple threads?

Reply:
I can't think of a valid use case where it would be needed to access it from multiple threads atm. The writing should only happen before initializing RUM, and the reading should only be done inside our RUM init code which is currently happening in a single thread. Though there's a valid case to be made on how far we should take "defensive programming" methodologies. I wonder if OTel has some rules on it.

Reply:
Yeah, most likely what could happen is someone iterating over the list and trying to mutate the list (removing by a specific type or something), it'd throw an exception, it'd not if its thread-safe (no iterable needed).
I don't see the need if not this use case as well since the single thread only during the init.